packages/version-bridge/src/lib/VersionedValue.ts

```diff
@@ -36,9 +36,27 @@ export function createVersionedValueMap<
   Versions extends { [version: number]: unknown },
 >(versions: Versions): VersionedValue<Versions> {
   Object.freeze(versions);
-  return {
+  const versionedValue: VersionedValue<Versions> = {
     atVersion(version) {
       return versions[version];
     },
   };
+  Object.defineProperty(versionedValue, '$value', {
+    configurable: false,
+    enumerable: true,
+    get() {
+      const highest = Object.keys(versions)
+        .map(v => parseInt(v, 10))
+        .sort((a, b) => b - a)[0];
+      return versions[highest];
```

Comment:
Not feeling this is super interesting tbh? Perhaps enough to just expose the map?

Reply:
I figured, since we usually have only 1 version, but it's removed now, it's certainly not _needed_.